charts/traffic/values.yaml

```diff
@@ -36,21 +34,13 @@ initContainers:
 
 # add extra environment params to app server
 extraEnv: |
-  - name: KEYCLOAK_REALM
```

Comment:
Why are those envs removed? Are they not used anymore?

Reply:
as I already showed you - the helm chart had duplicate values